aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -1,222 +1,276 @@
 
     [!code-html[Main](tutorial-high-frequency-realtime-with-signalr/samples/sample6.html?highlight=33-40)]
 
-    The above code moves the shape from the old location to the new one given by the server over the course of the animation interval (in this case, 100 milliseconds). Any previous animation running on the shape is cleared before the new animation starts.
-2. Start the application by pressing F5. Copy the page's URL, and paste it into a second browser window. Drag the shape in one of the browser windows; the shape in the other browser window should move. The movement of the shape in the other window should appear less jerky as its movement is interpolated over time rather than being set once per incoming message.
+1. Select the play button to start the application.
 
-    ![The application window](tutorial-high-frequency-realtime-with-signalr/_static/image8.png)
+1. Copy the page's URL.
 
-<a id="furthersteps"></a>
+1. Open another browser and paste the URL into the address bar.
 
-## Further Steps
+1. Drag the shape in one of the browser windows.
 
-In this tutorial, you've learned how to program a SignalR application that sends high-frequency messages between clients and servers. This communication paradigm is useful for developing online games and other simulations, such as [the ShootR game created with SignalR](https://shootr.azurewebsites.net/).
+The movement of the shape in the other window will appear less jerky. The app interpolates its movement over time rather than being set once per incoming message.
 
-The complete application created in this tutorial can be downloaded from [Code Gallery](https://code.msdn.microsoft.com/SignalR-20-MoveShape-Demo-6285b83a).
+This code moves the shape from the old location to the new one. The server gives the position of the shape over the course of the animation interval. In this case, that's 100 milliseconds. The app clears any previous animation running on the shape before the new animation starts.
 
-To learn more about SignalR development concepts, visit the following sites for SignalR source code and resources:
+## Additional resources
 
-- [SignalR Project](http://signalr.net)
-- [SignalR Github and Samples](https://github.com/SignalR/SignalR)
-- [SignalR Wiki](https://github.com/SignalR/SignalR/wiki)
+The communication paradigm you just learned about is useful for developing online games and other simulations, like [the ShootR game created with SignalR](https://shootr.azurewebsites.net/).
 
 For a walkthrough on how to deploy a SignalR application to Azure, see [Using SignalR with Web Apps in Azure App Service](../deployment/using-signalr-with-azure-web-sites.md). For detailed information about how to deploy a Visual Studio web project to a Windows Azure Web Site, see [Create an ASP.NET web app in Azure App Service](https://azure.microsoft.com/documentation/articles/web-sites-dotnet-get-started/).
```

Comment:
Is the Using SignalR with Web Apps tutorial on the list to be updated?  If not, it would be best to remove the links to it from the docs we're updating.

Reply:
The Using SignalR with Web Apps tutorial is not going to be updated for this User Story. I'll remove reference to it in this article and the other three that we've worked on.